### ISSUE TITLE
[question] Is it possible to set docker builder share intermediate levels and images with host's docker?

### ISSUE THREAD

**Issue by iav** [docker/buildx#201]
Is it possible to set docker builder share intermediate levels and images with host's docker?

**Comment by jsternberg** [comment]
I don't believe this is possible since buildkit doesn't necessarily create intermediate levels or images unless an export operation takes place. I'm going to close this since the issue is old but we can create a discussion if this is still relevant or if I'm wrong with my answer.
